yaksh/templates/yaksh/question.html

```diff
@@ -160,7 +160,7 @@
                                 <span> Files to download for this question </span> <hr>
                                 {% for f_name in files %}
                                   <div class="yakshwell">
-                                    <a href="{{f.file.url}}" class="btn btn-outline-secondary"><b>{{forloop.counter}}.</b> {{f_name.file.name}}</a>
+                                    <a href="{{f_name.file.url}}" class="btn btn-outline-secondary" target="_blank"><b>{{forloop.counter}}.</b> {{f_name.filename}}</a>
```

Comment:
It should be ```f_name.get_filename``` as you have added ```get_filename``` and not ```filename```
So kindly add a test case to avoid such error.